config/custom-environment-variables.yaml

```diff
@@ -112,3 +112,7 @@ ecosystem:
     store: ECOSYSTEM_STORE
     # Badge service (needs to add a status and color)
     badges: ECOSYSTEM_BADGES
+    # List of module names, or objects { name, config } for instantiation to use in sd-setup
+    setup: SD_SETUP_PLUGINS
```

Comment:
Actually sd-scm is the default config already (1st param that gets passed into Bookend constructor), so this will have no effects